test-app/article-controller.js

```diff
@@ -0,0 +1,27 @@
+import httpErrors from 'http-errors';
+import urls from '../src/utils/urls.js';
+import DocumentStore from '../src/stores/document-store.js';
+
+const { NotFound } = httpErrors;
+
+class ArticleController {
+  static get inject() { return [DocumentStore]; }
+
+  constructor(documentStore) {
+    this.documentStore = documentStore;
+  }
+
+  registerPages(router) {
+    router.get('/articles/*', async (req, res) => {
+      const slug = req.params[0] || '';
+      const doc = await this.documentStore.findOne({ namespace: 'articles', slug });
```

Comment:
I think now all documents have `namespace: article`, right?
If so, then maybe we should ditch it completely, from code and from the database.

Or is there still any reason to keep it?

Reply:
It's planned to get rid of the namespace altogether, we just didn't want to blow it up with that as well, same as with the image-tiles (we discussed during one coffee break when you were not here I think).

Reply:
Let's make a ticket for that then please and mention the remaining steps of removing this namespace.
I guess migration script for the DB index + migration for the data + code usage?

Reply:
Done: https://educandu.atlassian.net/browse/EDU-159, this issue is linked. I am not sure if there are further steps to do before we can completely remove the namespaces.